roles/contrail_node/tasks/main.yaml

```diff
@@ -0,0 +1,47 @@
+---
+- when:
+  - docker_registry is not defined
+
+  block:
+  - name: Untar Contrail tar-file
+    unarchive:
+      src: "{{ contrail_docker_images_path }}/contrail-kubernetes-docker-images_{{ contrail_version }}.tgz"
+      dest: "/tmp"
+
+  - name: Load Contrail-vRouter-Agent image
+    docker_image:
+      name: contrail-agent
+      tag: "{{ contrail_version }}"
+      timeout: 300
+      load_path: "/tmp/contrail-agent-{{ os_release }}-{{ contrail_version }}.tar.gz"
```

Comment:
Let's namespace this variable better.

contrail_os_release would be more appropriate.